app/models/dive.rb

```diff
@@ -0,0 +1,13 @@
+class Dive < ApplicationRecord
+  validates :style, presence: true, length: { minimum: 3 }
+  validates :number, presence: true
+  validates :depth, presence: true
+  validates :visibility, presence: true
+  validates :tempurature, presence: true
+  validates :longitude, presence: true
+  validates :latitude, presence: true
+  validates :dive_buddy, presence: true
+
```

Comment:
dive buddy, dpeth, and temp you may want to make optional. I dont know that every diver would have all of these, though you may know more
you also probably want to update `number` to be something more specific, like `x_number` since i dont know what it represents